api/src/main/java/io/grpc/ClientCall.java

```diff
@@ -141,6 +141,10 @@ public void onMessage(T message) {}
      * An additional block of trailer metadata may be received at the end of the call from the
      * server. An empty {@link Metadata} object is passed if no trailers are received.
      *
+     * <p>This method should not throw. If this method throws, there is no way to be notified of the 
```

Comment:
The CI is failing because both of these lines exceed 100 columns. Please rewrap these lines.